cmd/buildah/bud.go

```diff
@@ -303,3 +310,37 @@ func budCmd(c *cobra.Command, inputArgs []string, iopts budResults) error {
 	_, _, err = imagebuildah.BuildDockerfiles(getContext(), store, options, dockerfiles...)
 	return err
 }
+
+// discoverDockerfile tries to find a Dockerfile within the provided `path`.
+func discoverDockerfile(path string) (foundDockerfile string, err error) {
```

Comment:
I'm on the fence about the location of this function.  Kind of thinking that it should be in a utility.go type of file instead of here.  However, I'm going to push this through now to get the segv happy and we can revisit.  @nalind WDYT?